CMakeLists.txt

```diff
@@ -1,11 +1,20 @@
 # Check environment
 cmake_minimum_required(VERSION 3.5)
 
-# Set the project name and version
+# set the project name and version
 project(NEXUS
         VERSION 0.2.0
         LANGUAGES Fortran)
 
+# set compiler
+if(NOT DEFINED ENV{FC})
+  message(FATAL_ERROR "CMAKE_Fortran_COMPILER is not defined")
+endif()
+
+set(CMAKE_Fortran_COMPILER $ENV{FC})
+set(CMAKE_C_COMPILER $ENV{CC})
+set(CMAKE_CXX_COMPILER $ENV{CXX})
+
```

Comment:
This change is a reversion. CMake recognizes those env vars (`FC`, `CC`, etc.) automatically.